stdlib/testing/testdata/aggregate_window.flux

```diff
@@ -31,14 +33,12 @@ outData = "
 ,,1,2018-05-22T00:00:00Z,2018-05-22T00:01:00Z,2018-05-22T00:00:30Z,used_percent,disk,disk1s1,apfs,host.local,/tmp,35
 ,,1,2018-05-22T00:00:00Z,2018-05-22T00:01:00Z,2018-05-22T00:01:00Z,used_percent,disk,disk1s1,apfs,host.local,/tmp,45
 "
-
 aggregate_window = (table=<-) =>
-  table
-  |> range(start: 2018-05-22T00:00:00Z, stop:2018-05-22T00:01:00Z)
-  |> aggregateWindow(every:30s,fn:mean)
+	(table
+		|> range(start: 2018-05-22T00:00:00Z, stop: 2018-05-22T00:01:00Z)
+		|> aggregateWindow(every: 30s, fn: mean))
+
+test test_aggregate_window = () =>
```

Comment:
I'm ok with this keyword, but not the stutter.  `test aggregate_window = () => `  sounds better.  Many of the existing tests have a `t_` prefix to avoid naming collisions on function calls.

Reply:
I guess I'll have to go 1 by 1 and update the test names so that they're descriptive but do not conflict with any function names.